docs/source/science/NISAR/NISAR_access.ipynb

```diff
@@ -2,23 +2,23 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    from osgeo import gdal
</pre><p>Might fail if we're no longer installing osgeo</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/218/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<pre class="ql-syntax" spellcheck="false">Line #2.    from osgeo import gdal
</pre><p>Hmm, it actually still seemed to work when I ran through it. Should I just update it to import gdal like in the beginning of the tutorial? Or can that be removed, along with numpy since they were installed in the beginning?</p>

Reply:
We can keep the imports down below since that's the only place they're being used

I think that part of the problem is that we're installing using "!" instead of "%" so it's could be different environments? Maybe we switch this tutorial to use something like rasterio?

Reply:
oh, I see - I don't know if we need to import numpy up above if it's not being used until down below

Reply:
```
import numpy as np
import rasterio
import matplotlib.pyplot as plt


# Extract a subset of the SLC to display
x0 = 0
y0 = 10
x_offset = 1000
y_offset = 1000

with rasterio.open('HH.slc') as ds:
    # Define the window of data to read
    window = rasterio.windows.Window(x0, y0, x_offset, y_offset)
    
    # Read the data from the specified window
    slc = ds.read(1, window=window)
    print(slc)

fig = plt.figure(figsize=(14, 12))

# Display amplitude of the slc
ax = fig.add_subplot(2,1,1)
ax.imshow(np.abs(slc), vmin = -2, vmax=2, cmap='gray')
ax.set_title("amplitude")

# Display phase of the slc
ax = fig.add_subplot(2,1,2)
ax.imshow(np.angle(slc))
ax.set_title("phase")

plt.show()
```

Something like this, and including rasterio in the installs above when using "%"

Reply:
<p>Okay, it should be updated now </p>